Snowflake.Data.Tests/SFDbCommandBuilderTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿/*
+ * Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+namespace Snowflake.Data.Tests
+{
+    using NUnit.Framework;
+    using Snowflake.Data.Client;
+
+    [TestFixture]
+    class SFDbCommandBuilderTest : SFBaseTest
+    {
+        SnowflakeDbCommandBuilder builder;
+
+        [SetUp]
+        public void BeforeTest()
+        {
+            builder = new SnowflakeDbCommandBuilder();
+        }
+
+        [Test]
+        [Ignore("DbCommandBuilder")]
+        public void DbCommandBuilderDone()
+        {
+            // Do nothing;
+        }
+
+        [Test]
+        public void TestEmptyCommandBuilder()
+        {
+            Assert.AreEqual(null, builder.DataAdapter);
+            Assert.AreEqual(SnowflakeDbCommandBuilder.DEFAULT_QUOTE_PREFIX, builder.QuotePrefix);
+            Assert.AreEqual(SnowflakeDbCommandBuilder.DEFAULT_QUOTE_SUFFIX, builder.QuoteSuffix);
+        }
+
+        [Test]
+        public void TestCommandBuilderWithAdapter()
+        {
+            SnowflakeDbDataAdapter adapter = new SnowflakeDbDataAdapter();
+            builder = new SnowflakeDbCommandBuilder(adapter);
+            Assert.AreEqual(adapter, builder.DataAdapter);
+            Assert.AreEqual(SnowflakeDbCommandBuilder.DEFAULT_QUOTE_PREFIX, builder.QuotePrefix);
+            Assert.AreEqual(SnowflakeDbCommandBuilder.DEFAULT_QUOTE_SUFFIX, builder.QuoteSuffix);
+        }
+
+        [Test]
+        public void TestCommandBuilderSetPrefix()
+        {
+            string newQuotePrefix = "\'";
+
+            Assert.AreEqual(SnowflakeDbCommandBuilder.DEFAULT_QUOTE_PREFIX, builder.QuotePrefix);
```

Comment:
It sounds to me like testing a constructor - default quotation. If really needed I would put it into a separate test.

Reply:
You're right, it's already covered in that test. It's removed